test/include.test.js

```diff
@@ -1224,6 +1224,24 @@ describe('include', function() {
     });
   });
 
+  it('should not throw on fetch User if include is true', function(done) {
```

Comment:
Fairly minor, but for consistency's sake perhaps this test message could indicate that a boolean value should not throw an error, rather than just the value true.